src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -248,7 +249,23 @@ public SCMSourceCriteria getSCMSourceCriteria(SCMSource source) {
             criteriaTL.set(old);
         }
     }
-    
+
+    /**
+     * Will create an specialized view when there are no repositories found, which contain a Jenkinsfile
+     */
+    @Override
+    public View getPrimaryView() {
+        if (getItems().isEmpty()) {
+            return new EmptyOrganizationFolderView(this);
+        }
+        return super.getPrimaryView();
+    }
+
+    @Override
+    public AbstractFolderDescriptor getDescriptor() {
+        return new DescriptorImpl();
+    }
```

Comment:
It should return the correct descriptor without overriding (it's implemented [here](https://github.com/jenkinsci/cloudbees-folder-plugin/blob/9cbe993366137de106dbab2cfb7dfc51433324c8/src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java#L282)).